src/components/api-tokens/token.ts

```diff
@@ -0,0 +1,14 @@
+import { randomBytes } from 'crypto';
+
+export const DEFAULT_KEY_LENGTH = 8;
+export const DEFAULT_SECRET_LENGTH = 64;
+
+export function generateKey(prefix: string, name: string, length = DEFAULT_KEY_LENGTH): string {
+  const key = randomBytes(length * 2).toString('hex').slice(0, length);
+
+  return `${prefix}-${name}-${key}`;
+}
+
+export function generateSecret(length = DEFAULT_SECRET_LENGTH): string {
+  return randomBytes(length * 2).toString('base64').slice(0, length);
```

Comment:
```suggestion
  return randomBytes(length * 2).toString('hex').slice(0, length);
```

So we can avoid `=` `/` and `+`

Reply:
Damn, I started with hex, but wanted some characters in the token xD